toqito/rand/random_density_matrix.py

```diff
@@ -61,7 +61,7 @@ def random_density_matrix(
 
     >>> from toqito.rand import random_density_matrix
     >>> bures_mat = random_density_matrix(2, distance_metric="bures")
-    >>> bures_mat
+    >>> bures_mat # doctest: +SKIP
```

Comment:
And here too.

Reply:
Same as my other reply https://github.com/vprusso/toqito/pull/459#discussion_r1484922829